### PR TITLE
Send an `AnalyticsEvent` on `fidesctl init` execution

### LINKED ISSUE #420: Send an analytics event when executing `fidesctl init`
### Is your feature request related to a specific problem?

Analytics are not collected when users run the `fidesctl init` command.

### Describe the solution you'd like

When a user executes `fidesctl init`, an analytics event should be sent via the fideslog SDK.

### Describe alternatives you've considered, if any

We can do nothing, and continue to miss out on these precious analytics.

### Additional context

This may perhaps be made easier to implement if done after #393.

### PR DESCRIPTION
**Pull request by PSalant726**
Closes #420

### Code Changes

* [x] Clean up & clarify `init` command output
* [x] Modernize `init` command output references in documentation
* [x] Send an `AnalyticsEvent` on execution of `fidesctl init`

### Steps to Confirm

* [x] `make cli`
* [x] `fidesctl init`
* [x] Observe that a new analytics event is written to Snowflake

### Pre-Merge Checklist

* [x] All CI Pipelines Succeeded
* Documentation Updated:
  - [x] documentation complete, or draft/outline provided (tag docs-team to complete/review on this branch)
  - [ ] documentation issue created (tag docs-team to complete issue separately)
* [x] Issue Requirements are Met
* [x] Update `CHANGELOG.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,15 +12,15 @@
 
 ![Fidesctl banner](docs/fides/docs/img/fidesctl.png "Fidesctl banner")
 
- _[Join the waitlist](https://ethyca.com/waitlist/) to get started with our free, hosted version!_ 
+ _[Join the waitlist](https://ethyca.com/waitlist/) to get started with our free, hosted version!_
```
Comment (ThomasLaPiana) [inline README.md:15]: @iamkelllly is this still relevant 👀

**Review by ThomasLaPiana — APPROVED** [review]
LGTM! Just want to make sure that this won't conflict with the other docs changes ongoing

**Comment by ThomasLaPiana** [comment]
@conceptualshark will the docs changes here conflict with #531 ?

**Comment by PSalant726** [comment]
> will the docs changes here conflict with #531 ?

I removed the same block that was removed in #531. We should see a merge conflict, because both PRS modify the same lines, but the changes should be identical.

**Comment by ThomasLaPiana** [comment]
@PSalant726 appreciated, thanks for checking that. Will  merge and note this on that PR

**Pull request merged**